PhpAmqpLib/Channel/AbstractChannel.php

```diff
@@ -218,7 +218,7 @@ public function next_frame($timeout = 0)
     }
 
     /**
-     * @param $method_sig
+     * @param string $method_sig
```

Comment:
It must be a string.
`AbstractChannel#dispatch()` passes `$method_sig` into `AbstractChannel#$methodMap` methods. 

See https://github.com/php-amqplib/php-amqplib/blob/master/PhpAmqpLib/Helper/Protocol/MethodMap091.php specifically the `valid_method` and `get_method`.

Also `$method_sig` is "built" using `AbstractChannel#build_method_signature()` which returns a string.